docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -29,6 +29,20 @@ was chosen as it is the standard location for resources in `jar` files as define
 Quarkus can be used without Servlet following this convention allows existing code that places its resources in this
 location to function correctly.
 
+=== WebJar Locator Support
+
+If you add the `quarkus-webjars-locator` extension to your project, you do not need to specify version
+for webjars. That is, a URL like `/webjars/jquery/3.1.1/jquery.min.js` becomes `/webjars/jquery/jquery.min.js` in your
+HTML files. To use it, add the following to your project's dependencies:
+
+[source, xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-webjars-locator</artifactId>
+</dependency>
```

Comment:
I'd also add the jquery webjar itself here in the pom example too, for illustration:

```
    <dependency>
      <groupId>org.webjars</groupId>
      <artifactId>jquery</artifactId>
    </dependency>
```